core/src/test/java/berith/caym/crypto/SignTransactionTest.java

```diff
@@ -0,0 +1,38 @@
+package berith.caym.crypto;
+
+import java.math.BigInteger;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.web3j.crypto.ECKeyPair;
+import org.web3j.rlp.RlpEncoder;
+import org.web3j.rlp.RlpList;
+import org.web3j.utils.Numeric;
+
+import static org.hamcrest.CoreMatchers.is;
+
+public class SignTransactionTest {
+	
+	@Test
+	public void testSignTx() {
+		Transaction tx = new Transaction(
+			BigInteger.valueOf(1),
+			BigInteger.valueOf(1000000000l),
+			BigInteger.valueOf(21000),
+			"1234567890123456789012345678901234567890",
+			BigInteger.valueOf(1234567890l),
+			"",
+			1,
+			1
+		);

```

Comment:
need more test datas :)
how about use List<Tuple<Transaction,String,Exception>, where string is expected result ?

Reply:
There is no datas, data is the plural form of datum 🤣🤣